### DISCUSSION TITLE
is supabase-js library supported in non-node runtimes like vercel edge functions?

### DISCUSSION THREAD

**Discussion by GorvGoyl** [supabase/supabase#13161] [Questions] [answered]
is supabase-js library supported in non-node runtimes like vercel edge functions?

**Comment by GaryAustin1** [comment, marked as answer]
Supabase-js works in just plain javascript.

https://supabase.com/docs/reference/javascript/installing#install-via-cdn

And in Deno edge functions:
https://supabase.com/docs/reference/javascript/installing#use-at-runtime-in-deno
